src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -0,0 +1,69 @@
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonUtils;
+import org.photonvision.targeting.PhotonPipelineResult;
+
+import static frc.robot.Constants.Field.HUB_POSITION;
+
+public class Vision extends SubsystemBase {
+
+    private static final double CAMERA_HEIGHT_METERS = 1;
+    private static final double TARGET_HEIGHT_METERS = 2.64;
+    private static final double CAMERA_PITCH_RADIANS = Math.toRadians(45);
```

Comment:
Move to constants, and also this class shouldn't exist in this branch (so please delete it)